README.md

```diff
@@ -54,9 +54,10 @@
 
 # LTS Plan
 
-New semver-major releases of Node.js are cut from `master` every six months.
-New even-numbered versions (e.g. v6, v8, v10, etc) are cut in April. New 
-odd-numbered versions (e.g. v5, v7, v9) are cut in October.
+New semver-major releases of Node.js are cut from `master` every six months
+and become the "current" release.  New even-numbered versions (e.g.  v6, v8,
+v10, etc) are cut in April.  New odd-numbered versions (e.g.  v5, v7, v9)
+are cut in October.
```

Comment:
I think a small amount understand of information in here is valid in order to make it easy for people to understand what the term "current" means as opposed to LTS, and my current PR reflects that as there isn't much written about it anywhere at the moment.  There is the question of where you put any more information that is currently in this PR regarding the processes surrounding how things are selected and put into current

From a contributor perspective (and this PR came about as a result of an update I was making to CONTRIBUTOR.md) the process for getting things into current and LTS is broadly the same, but the selection process isn't.

@Fishrock123 Which documentation file in the main repository were you thinking of for that additional information? I don't want to make the contributor process any more complex than it needs to be by directing new contributors to too many different places (which may suggest that having it in CONTRIBUTING.md could be the main repo is the right place - I think it should be there or here for simplicity...)